Spring.FluentContext/IFluentConfigurableApplicationContext.cs

```diff
@@ -0,0 +1,9 @@
+using Spring.Objects.Factory.Config;
+
+namespace Spring.FluentContext
+{
+    public interface IFluentConfigurableApplicationContext
+    {
+        IConfigurableListableObjectFactory TemporaryInitObjectFactory { get; set; }
```

Comment:
I left my thoughts about this `TemporaryInitObjectFactory` above. It sounds like a hack exposing some internals to the public. I am sure it can be done differently.